projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.scss

```diff
@@ -33,8 +32,8 @@
     background: var(--wvr-dropdown-menu-background);
     border: var(--wvr-dropdown-menu-border);
     border-radius: var(--wvr-dropdown-menu-border-radius);
-    left: var(--wvr-dropdown-menu-x-offset);
-    margin-top: var(--wvr-dropdown-menu-y-offset);
+    left: var(--wvr-dropdown-menu-x-offset) !important;
```

Comment:
Ahhhhh. The oh so undesirable `!important`. Have all other approaches been exhausted?

Reply:
The other solution that were explored introduced too much additional css. This important was introduced to avoid re-implementing more of bootstraps styling.